docs/devGuide/development/workflow.md

```diff
@@ -190,6 +190,20 @@ After which, you can update the **expected** test files with: `npm run updatetes
   as uncommitted changes due to the way they are generated. If you are not directly modifying those files in your PR, you should **discard those changes** and **do not commit** them.
 </box>
 
+<box type="tip" seamless header="Solving merge conflicts in expected test files">
+
+1. Ensure the fork is latest
```

Comment:
Does "Ensure that the fork is synced with the upstream branch" work better?

Reply:
Thanks for the change! I think repo might be a better word than branch so its clearer :)